config/default.yml

```
@@ -1782,9 +1782,10 @@ Lint/DuplicateBranch:
   Description: Checks that there are no repeated bodies within `if/unless`, `case-when` and `rescue` constructs.
   Enabled: pending
   VersionAdded: '1.3'
-  VersionChanged: '1.7'
+  VersionChanged: '<<next>>'
```

Although a configuration parameter has been added, the default behavior has not changed, so updating `VersionChanged` is not necessary.

```suggestion
  VersionChanged: '1.7'
```